### PR TITLE
Ably branding

### PR DESCRIPTION
**Pull request by AmirQSD**
Use Ably branding for demo app

### CODE REVIEW

**Review by owenpearson — CHANGES_REQUESTED** [review]
Looks much better! I think one thing which looks a bit odd to me is the styling for the chat container. I think it would look better with a border and some padding (about 0.5rem).

I had a little play around and found that adding these styles to .tab-content looks better:
```css
border-left-style: solid;
padding: 0.5rem;
border-right-style: solid;
border-left-width: 1px;
border-right-width: 1px;
border-bottom-width: 1px;
border-bottom-style: solid;
border-bottom-right-radius: 1rem;
border-bottom-left-radius: 1rem;
border-color: lightgray;
```

File: routes/web.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('listen');
```
Comment (sacOO7) [inline routes/web.php:22]: We can rename to `return view('ChatRoom');`

File: routes/web.php [hunk 2]
```diff
@@ -37,8 +37,8 @@
     event(new PresenceMessageNotification($request->channel, $request->message));
 });
 
-Route::get('/listen', function () {
-    return view('listen');
-})->name('listen');
+Route::get('/ChatRoom', function () {
+    return view('ChatRoom');
+})->name('ChatRoom');
```
Comment (qsdstefan) [inline routes/web.php:42]: This route is now unused, can we remove it?

Comment (AmirQSD) [reply routes/web.php:42]: Yes, makes sense. We can do it, I will make a commit.

Comment (sacOO7) [reply routes/web.php:42]: Thanks

**Review by sacOO7 — CHANGES_REQUESTED** [review]
Add one more SS for `homepage`

**Comment by sacOO7** [comment]
> > Can you add one more screenshot under README, saying `homepage`
> 
> @sacOO7 FYI the homepage now shows the Chat box straight away, we removed the default Laravel welcome page

Ohkay, maybe you can include one SS for `Sign up`

**Comment by qsdstefan** [comment]
> Ohkay, maybe you can include one SS for `Sign up` screen

Updated README.md as per suggestions

**Review by sacOO7 — CHANGES_REQUESTED** [review]
Rename `Register` button to `Sign up`. Align it to the center horizontally.

**Review by sacOO7 — CHANGES_REQUESTED** [review]
`Sign up` title can be called `User Registration`. 
I think no need to make changes to `Sign up` button, just slight top and bottom margin can be added

**Review by sacOO7 — CHANGES_REQUESTED** [review]
@qsdstefan forgot to add, can you add those missing steps @owenpearson mentioned sometime back? It should be really easy for any non-laravel dev to do the setup without facing any issues.

**Comment by qsdstefan** [comment]
@AmirQSD FYI please see the second point Sachin posted above, as you're currently working on customizing the alert popups/input prompts.

**Review by sacOO7 — APPROVED** [review]
Thanks for updating the app as per the review changes.
The app looks way better : )

**Review by owenpearson — APPROVED** [review]
Looks awesome, great work on this 🙂

**Review by qsdstefan — APPROVED** [review]
LGTM!

**Pull request merged**